### PR TITLE
RHDEVDOCS-3771 - Fluentd to collector

### PR DESCRIPTION
**Pull request by libander**
Version(s): 4.9+

Issue: https://issues.redhat.com/browse/RHDEVDOCS-3771

Link to docs preview:

QE review: @anpingli / @QiaolingTang / @kabirbhartiRH 
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/cluster-logging-manual-rollout-rolling.adoc [hunk 1]
```diff
@@ -31,11 +31,11 @@ $ oc project openshift-logging
 $ oc get pods -l component=elasticsearch-
 ----
 
-. Scale down the Fluentd pods so they stop sending new logs to Elasticsearch:
+. Scale down the collector pods so they stop sending new logs to Elasticsearch:
 +
 [source,terminal]
 ----
-$ oc -n openshift-logging patch daemonset/logging-fluentd -p '{"spec":{"template":{"spec":{"nodeSelector":{"logging-infra-fluentd": "false"}}}}}'
+$ oc -n openshift-logging patch collector -p '{"spec":{"template":{"spec":{"nodeSelector":{"logging-infra-collector": "false"}}}}}'
```
Comment (kabirbhartiRH) [inline modules/cluster-logging-manual-rollout-rolling.adoc:38]: This should be `oc -n openshift-logging patch daemonset /collector`

File: modules/cluster-logging-manual-rollout-rolling.adoc [hunk 2]
```diff
@@ -213,9 +213,9 @@ $ oc exec elasticsearch-cdm-5ceex6ts-1-dcd6c4c7c-jpw6 -c elasticsearch -- es_uti
 }
 ----
 
-. Scale up the Fluentd pods so they send new logs to Elasticsearch.
+. Scale up the collector pods so they send new logs to Elasticsearch.
 +
 [source,terminal]
 ----
-$ oc -n openshift-logging patch daemonset/logging-fluentd -p '{"spec":{"template":{"spec":{"nodeSelector":{"logging-infra-fluentd": "true"}}}}}'
+$ oc -n openshift-logging patch collector -p '{"spec":{"template":{"spec":{"nodeSelector":{"logging-infra-collector": "true"}}}}}'
```
Comment (kabirbhartiRH) [inline modules/cluster-logging-manual-rollout-rolling.adoc:220]: Same as above.

**Review by deerskindoll — APPROVED** [review]
lgtm

**Pull request merged**